marlowe-playground-client/entry.js

```diff
@@ -1,3 +1,5 @@
+/*eslint-env node*/
+/*global global*/
```

Comment:
what's this?

Reply:
Huh. Wasn't me. Has it been removed from master since I rebased?

Reply:
I think I use eslint because I use VSCode 🤔  Maybe we should have a CI job though?